### PR TITLE
VEIOSS-647 | Remove 'Rejoin This Service' Link from Dashboard When Tr…

### PR DESCRIPTION
**Pull request by Lee-Powell**
…ader Has Deregistered From VAT

### CODE REVIEW

**Review by hmrcvmu — DISMISSED** [review]

File: app/connectors/RegistrationConnector.scala [hunk 1]
```diff
@@ -38,4 +39,7 @@ class RegistrationConnector @Inject()(config: Configuration, httpClientV2: HttpC
   def getAccounts()(implicit hc: HeaderCarrier): Future[EACDEnrolments] =
     httpClientV2.get(url"$baseUrl/accounts").execute[EACDEnrolments]
 
+  def getVatCustomerInfo()(implicit hc: HeaderCarrier): Future[VatCustomerInfoResponse] =
```
Comment (DevAndyroo) [inline app/connectors/RegistrationConnector.scala:42]: we shouldn't need this endpoint as the get registration is a Wrapper of vatInfo and registration and therefore you should be able to get it from doing `request.registrationWrapper.vatInfo` from the request in context https://github.com/hmrc/ioss-returns-frontend/pull/129/files#diff-c9a12a9b7b7afc7e49545de41178ddf64898c92e777c20264d084803342c721aR128

File: app/views/YourAccountView.scala.html [hunk 2]
```diff
@@ -99,15 +113,41 @@ <h1 class="govuk-heading-l">
...
                                 @if(exclusion.quarantine) {
-                                    @govukWarningText(WarningText(
-                                        iconFallbackText = Option(messages("site.warning")),
-                                        content = Text(messages("yourAccount.quarantined", exclusion.displayRejoinDate))
-                                    ))
+                                    @if(hasDeregisteredFromVat) {
+                                        <div class="govuk-warning-text">
+                                            <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+                                            <strong class="govuk-warning-text__text">
+                                                <span class="govuk-warning-text__assistive">Warning:</span>
```
Comment (DevAndyroo) [inline app/views/YourAccountView.scala.html:120]: Why can't we use `govukWarningText` anymore?

**Review by DevAndyroo — CHANGES_REQUESTED** [review]

File: app/connectors/RegistrationConnector.scala (hunk 1 above)
Comment (Lee-Powell) [reply app/connectors/RegistrationConnector.scala]: Made changes used registrationWrapper.vatInfo

Comment (Lee-Powell) [reply app/connectors/RegistrationConnector.scala]: Made Changes used registrationWrapper.vatInfo

File: app/views/YourAccountView.scala.html (hunk 2 above)
Comment (Lee-Powell) [reply app/views/YourAccountView.scala.html]: Made changes reverted back to govukWarningText, not sure why I did it this way

**Review by DevAndyroo — DISMISSED** [review]

**Comment by Lee-Powell** [comment]
retest this please

**Comment by Lee-Powell** [comment]
retest this please

**Comment by Lee-Powell** [comment]
retest this please

**Comment by Lee-Powell** [comment]
retest this please

**Review by DevAndyroo — APPROVED** [review]

**Pull request merged**